Detectors/TRD/macros/calibrateKr.C

```diff
@@ -0,0 +1,38 @@
+#include "TRDCalibration/KrCalibration.h"
+
+using namespace o2::trd;
+
+void calibrateKr( TString sOpenFile )
+{
+    KrCalibration* krCalibration = new KrCalibration();
+
+    TFile* file = TFile::Open( sOpenFile );
+    TTree* tree = (TTree*) file->Get("nt_Krypton");
+    LocalGainFactor calObject; 
+    TH2F* hDet[540];
+
+    for( int idet = 0; idet < 540; idet++ ) {
+        TH2F* hDetDef = krCalibration->GetDetectorMap( tree, idet);
+        TH2F* hDetFilled = krCalibration->FillTheMap(hDetDef);
+        hDet[idet] = krCalibration->CreateNormalizedMap(hDetFilled);
+        for(int irow = 0; irow < hDet[idet]->GetNbinsY(); irow++ ) {
+            for(int icol = 0; icol < hDet[idet]->GetNbinsX(); icol++ ) {
+                float relativeGain = hDet[idet]->GetBinContent(hDet[idet]->GetXaxis()->FindBin(icol), hDet[idet]->GetYaxis()->FindBin(irow));
+                calObject.setPadValue(idet, icol, irow, relativeGain);
+            }
+        }
+    }
+
+    ///write map to CCDB
+    o2::ccdb::CcdbApi ccdb;
+    ccdb.init("http://ccdb-test.cern.ch:8080");
+    // o2::base::NameConf::getCCDBServer()
+    std::map<std::string, std::string> metadata;
+
+    ///upload temporarily
+    auto timeStamp = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count();
```

Comment:
OK, will do that. For now I keep them hardcoded and set starting date as the (afaik) first date with Kr run and end date as Jan 1, 2040.
I tried to create a function which would take a std::string with date and time and create the stamp from there but I keep getting unexpected dates this way. This will be resolved but for now the priority is to have something that functions the way I want it to.

Reply:
Ok. As long as the date is in the .C macro that's fine. Might be good to have that under version control anyways, rather than relying on remembering the right parameters after the next krypton run.